src/components/App.tsx

```diff
@@ -1277,6 +1281,20 @@ class App extends React.Component<ExcalidrawProps, AppState> {
     this.setState({ toastMessage: null });
   };
 
+  restoreFileFromShare = async () => {
+    const keys = await caches.keys();
+    const mediaCache = await caches.open(
```

Comment:
Just use `"web-share-target"` as the cache name. The filtering is not needed (it was needed in the other implementation).